cmd/skaffold/app/cmd/runner.go

```diff
@@ -35,6 +35,7 @@ import (
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/parser"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/runner"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/runner/runcontext"
+	v1 "github.com/GoogleContainerTools/skaffold/pkg/skaffold/runner/v1"
```

Comment:
do we still want to reference this as `runner`? or `runnerV1`?

Reply:
Thank you. Done

Reply:
The `make linters` fails if we don't use the alias "v1".

more context: golangci-lint has local-prefix enabled. https://github.com/golangci/golangci-lint/issues/209